### PR TITLE
Adjust project list status badge and text, fixed stories issue as well

### PR DESCRIPTION
**Pull request by tylerlov**

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-7)

File: cypress/fixtures/projects.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "id": "6d5fff43-d691-445d-a41a-7d0c639080e6",
     "numIssues": 73,
     "numEvents24h": 7,
-    "status": "error",
+    "status": "critical",
```
Comment (jkettmann) [inline cypress/fixtures/projects.json]: The changes in this file are likely the problem. This represents the API data. And neither `critical` nor `stable` exist in the API response. So if you revert these changes I guess the Cypress tests will work as well

File: features/projects/components/project-card/project-card.tsx [hunk 2]
```diff
@@ -50,7 +56,9 @@ export function ProjectCard({ project }: ProjectCardProps) {
             <div className={styles.issuesNumber}>{numEvents24h}</div>
           </div>
           <div className={styles.status}>
-            <Badge color={statusColors[status]}>{capitalize(status)}</Badge>
+            <Badge color={statusColors[status]}>
+              {capitalize(statusMappings[status])}
```
Comment (jkettmann) [inline features/projects/components/project-card/project-card.tsx]: There's no need for the capitalize function anymore if you simply capitalize the values in the `statusMappings` object. Apart from this the PR looks great. Well done 👍

Comment (tylerlov) [reply features/projects/components/project-card/project-card.tsx]: Thanks! Wasn't sure if the call to capitalize was there for any unknown/future reason, will clean that up

File: cypress/fixtures/projects.json (hunk 1 above)
Comment (tylerlov) [reply cypress/fixtures/projects.json]: Got it! Was thinking about this wrong when I made that change, thanks!
